CMakeLists.txt

```diff
@@ -24,7 +24,9 @@ option(BUILD_PYTHON "Build python bindings" ON)
 option(BUILD_LMDB "Build LMDB readers" OFF)
 option(BUILD_TENSORFLOW "Build TensorFlow plugin" OFF)
 option(BUILD_JPEG_TURBO "Build with libjpeg-turbo" ON)
-option(BUILD_NVJPEG "Build with nvJPEG" ON)
+option(BUILD_NVJPEG "Build with nvJPEG support" ON)
```

Comment:
Please update README as well.

Reply:
This was supposed to go in another PR, I added here by mistake.
I will adjust readme there.